src/index.js

```diff
@@ -1,16 +1,25 @@
 /* eslint-disable no-restricted-globals */
+import fetchScore from './modules/fetchScore.js';
 import './style.css';
-import lists from './modules/list.js';
+import postScore from './modules/postScore.js';
 
+const btnRefresh = document.getElementById('btn-refresh');
+const form = document.querySelector('form');
 function component() {
   const element = document.createElement('div');
-  lists();
+  fetchScore();
   return element;
 }
```

Comment:
- [ ] Please kindly use the arrow function instead of the `function` keyword as per requirements